### ISSUE TITLE
mention that components are in a separate repo (and provide insight into why)

### ISSUE THREAD

**Issue by travi** [dsmjs/site#481]

**Comment by micleners** [comment]
Can you add more detail here?

I was wondering about this from the PR feedback on [404 page](https://github.com/dsmjs/site/pull/471).

Would you envision a component in `components` be built to style the 404 page or will we start to build components in `site` too, for new layouts?

**Comment by travi** [comment]
the intend is for anything visible as a component would be in the components package. the site would need to import from that package into whatever is necessary to wire the component into gatsby or pass it the results of the data query.

in the 404 example, the site side would mostly just import the component and render it, but the actual visual parts should come from the package. i'm fine with merging something simple directly into the site to start with, but would want us to follow that up to extract the actual component.

does that help?

**Comment by micleners** [comment]
Yes, I think this process makes sense. Explaining this for folks coming on to contribute is a good idea (since this paradigm was not readily apparent to me) 👍I support this issue
